sample/src/main/java/org/aerogear/graphqlandroid/data/UserData.kt

```diff
@@ -107,6 +134,16 @@ class UserData(val context: Context) {
         val mutation = CreateTaskMutation.builder().title(title).description(description).build()
 
         val customCallback = object : ResponseCallback {
+            override fun onConflictDetected(
+                serverState: String,
+                clientState: String
+            ): Mutation<Operation.Data, Any, Operation.Variables> {
+                /*
+                 This would never be called as createTasks never causes conflicts
+                 */
+                return mutation as com.apollographql.apollo.api.Mutation<Operation.Data, Any, Operation.Variables>
```

Comment:
return null instead if you don't want to handle conflicts.

Reply:
I tried it, but it's not taking null here.